mod-fqm-manager/src/main/resources/corsair/mod-fqm-manager/fqm-junit.feature

```diff
@@ -15,27 +15,56 @@ Feature: mod-fqm-manager integration tests
+      | name                                                        |
+      | 'acquisitions-units.units.collection.get'                   |
+      | 'addresstypes.collection.get'                               |
+      | 'addresstypes.item.post'                                    |
+      | 'circulation-storage.loan-policies.collection.get'          |
+      | 'circulation-storage.loans.item.post'                       |
+      | 'circulation.loans.collection.get'                          |
+      | 'configuration.entries.collection.get'                      |
+      | 'departments.collection.get'                                |
+      | 'finance.exchange-rate.item.get'                            |
+      | 'inventory-storage.call-number-types.collection.get'        |
+      | 'inventory-storage.contributor-name-types.collection.get'   |
+      | 'inventory-storage.contributor-types.collection.get'        |
+      | 'inventory-storage.holdings.item.get'                       |
+      | 'inventory-storage.holdings.item.post'                      |
+      | 'inventory-storage.instance-statuses.collection.get'        |
+      | 'inventory-storage.instance-types.item.post'                |
+      | 'inventory-storage.instances.item.get'                      |
+      | 'inventory-storage.instances.item.post'                     |
+      | 'inventory-storage.items.item.get'                          |
+      | 'inventory-storage.items.item.post'                         |
+      | 'inventory-storage.loan-types.item.post'                    |
+      | 'inventory-storage.location-units.campuses.item.post'       |
+      | 'inventory-storage.location-units.institutions.item.post'   |
+      | 'inventory-storage.location-units.libraries.collection.get' |
+      | 'inventory-storage.location-units.libraries.item.post'      |
+      | 'inventory-storage.locations.collection.get'                |
+      | 'inventory-storage.locations.item.post'                     |
+      | 'inventory-storage.material-types.collection.get'           |
+      | 'inventory-storage.material-types.item.post'                |
+      | 'inventory-storage.modes-of-issuance.collection.get'        |
+      | 'inventory-storage.service-points.collection.get'           |
+      | 'inventory-storage.statistical-code-types.collection.get'   |
+      | 'inventory-storage.statistical-codes.collection.get'        |
+      | 'orders-storage.po-lines.item.post'                         |
+      | 'orders-storage.purchase-orders.item.post'                  |
+      | 'orders.item.get'                                           |
+      | 'orders.po-lines.item.get'                                  |
+      | 'organizations-storage.categories.collection.get'           |
+      | 'organizations-storage.organization-types.collection.get'   |
+      | 'organizations-storage.organizations.item.post'             |
+      | 'organizations.organizations.item.get'                      |
```

Comment:
```suggestion
      | 'organizations.organizations.collection.get'                      |
```

Reply:
I'm about to merge something that changes the org item permission to org collection, that's why I'm suggesting this